tools/Dockerfile

```diff
@@ -0,0 +1,34 @@
+FROM debian:stable-slim
+
+ENV DEBIAN_FRONTEND=noninteractive
+
+#This is required fo bypassing the tzdata installation
+ENV TZ=Europe/Bucharest
+RUN ln -snf /usr/share/zoneinfo/$TZ /etc/localtime && echo $TZ > /etc/timezone
+
+RUN apt-get update
+RUN apt-get install -y sudo
+RUN apt-get install -y git
+RUN apt-get install -y python
+RUN apt-get install -y curl
+RUN apt-get install -y lsb-release
+RUN apt-get install -y vim
+RUN apt-get install -y ninja-build
+RUN apt-get install -y pkg-config
+RUN apt-get install -y gcc-arm-linux-gnueabihf
+RUN apt-get install -y g++-8-arm-linux-gnueabihf
+RUN apt-get install -y ca-certificates 
+
+#Install JAVA
+RUN mkdir -p /usr/share/man/man1
+WORKDIR /opt
+RUN curl -L -o openjdk.tar.gz https://download.java.net/java/GA/jdk11/13/GPL/openjdk-11.0.1_linux-x64_bin.tar.gz
+RUN mkdir jdk
+RUN tar zxf openjdk.tar.gz -C jdk --strip-components=1
+RUN ln -sf /opt/jdk/bin/* /usr/local/bin/
+
+# clone react-native-webrtc repository
+WORKDIR /
+RUN git clone https://github.com/react-native-webrtc/react-native-webrtc.git
```

Comment:
This will only allow you to grab master.... I'd make the org/repo and then the SHA1 arguments that get substituted in so that someone can easily override it in a fork/testing on a different branch... if the fork/branch isnt public thats a problem they would have to overcome.

I get it, you're not aiming for automated builds of libwebrtc with this docker image.... you're using it as an image you can exec into and then run the commands you want.... I still think this would be nice, and would inherently allow for automated builds...

I'd probably go and clone down jitsi/webrtc (configurable) so you can point your tools command to that directory

Reply:
This is Titus' first ever Dockerfile, go easy on him ;-)

Currently the plan is to just setup the environment, run the setup phase and add the jitsi/webrtc origin.

Then building the library is a couple of commands away.

Let's see how this plays out, I'll be happy to see it extended further after we lay down the first stone.

Reply:
I didn't mean to come across harshly :)  sorry if it did!

Reply:
:) It's fine. I actually thought about passing the branch for react-native-webrtc, but not sure if that is worth it, since that one is used only for pulling the scripts, on which not a lot of further work is required. For the jitsi/webrtc repo, that would definitely be the case, but so far I was not able to find a solution to add those commands to the docker file. If I try to do any of the setup steps here, crazy permissions and other errors are thrown.